clis/daemon-common/src/daemon.rs

```diff
@@ -0,0 +1,122 @@
+//! Code for general daemon handling utilities and abstracting away related dependencies.
+//!
+
+cfg_if::cfg_if! {
+    if #[cfg(unix)] {
```

Comment:
Instead of disabling the code we could configure the manifest file to only support this target

Reply:
this also for comms.rs file

Reply:
That would be great, unfortunately I have no idea how to do that, this is the simplest thing I found :/
Do you have some resources on this topic?

Reply:
@lcruz99 It doesn't work for me too well, maybe I'm doing it wrong somehow, but it seems that it still tries to compile the dependencies and fails on the last step, which is not that much better. Also, this PR is not really about TCLI, so I guess it would be better to make another issue for improving this. The `daemon-common` crate doesn't need this conditions if we exclude it in windows testing (as it is unix-only crate). And for `teliod` maybe the cleanest way would be making separate files with OS-specific implementation, what do you think?

Reply:
Oh, in the end I just removed that conditions, as the code is cleaner and CI works with that